src/steelshield/unreal-engine-plugin/tokenprovider.md

```diff
@@ -0,0 +1,33 @@
+# Token Provider
+
+The Token Provider is the system responsible for acquiring or generating the *JWT* which is issued to the *SteelShield
+Token Service*.
+
+There are 2 Token Provider implementations which are supplied with the *SteelShield Plugin*, but the plugin integrator
+is able to develop their own Token Provider that works with their backend should they have a custom backend solution.
+
+***
+**NOTE**
+
+> It is advised that as much of the authentication and JWT generation as 
+> possible is done on the game backend server so that attackers are not 
+> able to reverse engineer and extract certificates on the client side 
+> in order to generate valid tokens that can be used in an attack. 
+
+***
```

Comment:
```suggestion
> [!CAUTION]
> It is advised that as much of the authentication and JWT generation as 
> possible is done on the game backend server so that attackers are not 
> able to reverse engineer and extract certificates on the client side 
> in order to generate valid tokens that can be used in an attack. 
```

This might be more suitable here, wdyt.

https://vitepress.dev/guide/markdown#github-flavored-alerts

Reply:
Thanks, thought I had already done all of these, guess I missed this one